core/api/make.cpp

```diff
@@ -387,8 +387,6 @@ namespace fc::api {
         }},
         // TODO(turuslan): FIL-165 implement method
         .NetAddrsListen = {},
-        // TODO(turuslan): FIL-165 implement method
-        .PaychVoucherAdd = {},
```

Comment:
Initialize all payment channel methods, at least default value `{}`, otherwise gcc will not compile

Reply:
Fixed, but:
<img width="578" alt="Screenshot 2020-06-29 at 09 41 44" src="https://user-images.githubusercontent.com/495478/85981352-c06cd600-b9ec-11ea-8d4b-f842e05bcca7.png">